packages/preferences/src/browser/preferences-tree-widget.ts

```diff
@@ -176,6 +176,9 @@ export class PreferencesContainer extends SplitPanel implements ApplicationShell
         if (this.treeWidget) {
             this.treeWidget.activate();
         }
+        if (this.currentEditor) {
+            this.currentEditor.activate();
```

Comment:
Should not only one widget get focus? For the code, it looks like the preference tree supposed to get a focus but does not get it for some reasons.

Reply:
with this change, the preference tree gets focus first, and then the editor gets focus. Is there any problem with this? to be honest i don't understand why the preference tree is supposed to get focus :) if there are no particular reason behind it, i will remove line 176-178.

Reply:
@akosyakov  does the following change look better ? Thank you !
```ts
if (this.currentEditor) {
    this.currentEditor.activate();
} else if (this.treeWidget) {
    this.treeWidget.activate();
}
```

Reply:
OK I tested a few combinations. I don't think activating the treeWidget was helpful. Please correct me if I missed anything.